platform-includes/getting-started-use/javascript.nestjs.mdx

```diff
@@ -94,3 +94,6 @@ export class ExampleExceptionFilter extends BaseExceptionFilter {
   }
 }
 ```
+
+By default, only unhandled exceptions that are not caught by an error filter are reported to Sentry.
+`HttpException`s also are not captured by default because they mostly act as control flow vehicles.
```

Comment:
l: let's maybe mention the derivatives of HttpException as there a quite a few
```suggestion
`HttpException`s (including [derivatives](https://docs.nestjs.com/exception-filters#built-in-http-exceptions)) are also not captured by default because they mostly act as control flow vehicles.
```